test/unit/output_generator_test.rb

```diff
@@ -10,7 +10,9 @@ class OutputGeneratorTest < ActiveSupport::TestCase
       test 'empty password is replaced by fake value so that virt-who runs in an non-interactive mode' do
         config.hypervisor_password = nil
         config.hypervisor_type = 'libvirt'
-        assert_includes output, 'virt-who-password --password \'' + OutputGenerator::LIBVIRT_FAKE_PASSWORD + '\''
+        fake_output = generator.virt_who_output
+        # (echo bGlidmlydF9mYWtlX3Bhc3N3b3Jk|base64 -d) = OutputGenerator::LIBVIRT_FAKE_PASSWORD
+        assert_includes fake_output, '$(echo bGlidmlydF9mYWtlX3Bhc3N3b3Jk|base64 -d)'
```

Comment:
```
assert_includes output, 'virt-who-password --password $(echo bGlidmlydF9mYWtlX3Bhc3N3b3Jk|base64 -d)'

Reply:
@lfu I can't use output because it's already declared here, it works fine for the other tests but not this one since we are altering the data after the fact.

https://github.com/theforeman/foreman_virt_who_configure/blob/2a03ccc962b128dc55236b1fb5fb9f078b333d9a/test/unit/output_generator_test.rb#L6